src/python/WMComponent/JobAccountant/AccountantWorker.py

```diff
@@ -212,16 +213,29 @@ def didJobSucceed(self, jobReport):
         if not jobReport.taskSuccessful():
             return False
 
-
         return True
 
     def isTaskExistInFWJR(self, jobReport, jobStatus):
         """
-        TODO: fix it, if it doesn't exist
+        If taskName is not available in the FWJR, then tries to
+        recover it getting data from the SQL database.
         """
         if not jobReport.getTaskName():
-            msg = "Report to developers, Investigate currupted fwjr for %s job id %s" % (jobStatus, jobReport.getJobID())
-            raise AccountantWorkerException(msg)
+            logging.error("Found a corrupted FWJR for a %s job with job id %s" % (jobStatus,
```

Comment:
maybe we can just make this warning  instead of error and say that "trying to recover"